src/Util/FileDumper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * BigBlueButton open source conferencing system - https://www.bigbluebutton.org/.
+ *
+ * Copyright (c) 2016-2024 BigBlueButton Inc. and by respective authors (see below).
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 3.0 of the License, or (at your option) any later
+ * version.
+ *
+ * BigBlueButton is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with BigBlueButton; if not, see <https://www.gnu.org/licenses/>.
+ */
+
+namespace BigBlueButton\Util;
+
+class FileDumper
+{
+    private const LINE   = "------------------------------------------------------------------------------------------------------------------------\n";
+    private const FOLDER = './var/dump/';
+
+    /**
+     * @param mixed $var
+     */
+    public static function DUMP($var): void
```

Comment:
The API library has currently no access to the file system. Dealing with file system exposes the end platform to several security risks. Where do we need this in the library? If it is intended for debugging I suggest using `tracy` library in require-dev.

Reply:
yes, the only reason was debugging. 
I can remove the class (and the`var`in `.gitignore` if you don't like it. And I do not know `tracy`... that maybe the reason i needed that class ;-)

Reply:
If we really need debugging for dev, I suggest adding https://github.com/nette/tracy to dev dependencies. `FileDumper` **must** be removed (sorry to say like it that) as it exposes too many risks.